app/src/main/java/com/github/andrezimmermann/gluappsample/client/mvp/event/ClickHandler.java

```diff
@@ -0,0 +1,10 @@
+package com.github.andrezimmermann.gluappsample.client.mvp.event;
+
+/**
+ * TODO: add javadoc
+ */
+public interface ClickHandler {
```

Comment:
Android already has a very similar interface, why you had to create another one?